v2/test/Makefile

```diff
@@ -31,5 +31,5 @@ tmp/context.tar.gz:
 # available in gnutar.
 	mkdir -p tmp
 # cd $(REPO_ROOT) && git archive --format=tar $$(git stash create) | gzip >v2/test/tmp/context.tar.gz
-# TODO(Bobgy): figure out how to allow archiving dirty region.
+# TODO(Bobgy): figure out how to allow archiving dirty folder.
```

Comment:
Perhaps just archive it using `tar -zcf` ?

Reply:
it will unnecessarily include all the files we ignore through `.gitignore`, and there are a lot of those files -- including the current tar.gz you are going to create, so it'll fail.

Reply:
> including the current tar.gz you are going to create, so it'll fail.
This is trivially fixable by creating the tar.gz in a different location. (P.S. I'm not sure it will actually fail.)

Reply:
Yes, I got that error when trying it.
Besides, I don't want the tar command to include the many other ignored files which can be huge in size, including node_modules, etc